certified-connectors/BuildingMindsTwin/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2128 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "BuildingMinds DigitalTwin Core",
+    "version": "1.0",
+    "description": "Connector to BuildingMinds DigitalTwin Core",
```

Comment:
@duncanbinger Please provide a valid and more detailed description for your connector.

Reply:
@sriyen-msft description updated accordingly.